### DISCUSSION TITLE
How can I contribute to translation?

### DISCUSSION THREAD

**Discussion by Archer-Tatsu** [seerr-team/seerr#621] [General]
I have finished reading this page https://github.com/Fallenbagel/jellyseerr/blob/develop/CONTRIBUTING.md and found the part about translation. However, the provided link, https://jellyseerr.borgecube.de/engage/jellysseerr/, is not available.

**Comment by fallenbagel** [comment]
Oh I messed up. The link is:
https://jellyseerr.borgcube.de/projects/jellyseerr/jellyseerr-frontend/
